README.md

```diff
@@ -1,25 +1,24 @@
-# LaunchDarkly Sample .NET Application 
+# LaunchDarkly sample .NET application
 
 We've built a simple console application that demonstrates how LaunchDarkly's SDK works.
 
-Below, you'll find the basic build procedure, but for more comprehensive instructions, you can visit your [Quickstart page](https://app.launchdarkly.com/quickstart#/) or the [.NET SDK reference guide](https://docs.launchdarkly.com/sdk/server-side/dotnet).
-
-## Build instructions 
+Below, you'll find the build procedure. For more comprehensive instructions, you can visit your [Quickstart page](https://app.launchdarkly.com/quickstart#/) or the [.NET SDK reference guide](https://docs.launchdarkly.com/sdk/server-side/dotnet).
 
 This is a .NET Core application that can be built on any platform where .NET is available. It can be run either in Visual Studio, or from the command line if you have installed the .NET command-line tools.
 
-1. Edit `HelloDotNet/Hello.cs` and set the value of `SdkKey` to your LaunchDarkly SDK key. If there is an existing boolean feature flag in your LaunchDarkly project that you want to evaluate, set `FeatureFlagKey` to the flag key.
+## Build instructions
 
-```csharp
-    public const string SdkKey = "1234567890abcdef";
+1. Set the environment variable `LAUNCHDARKLY_SERVER_KEY` to your LaunchDarkly SDK key. If there is an existing boolean feature flag in your LaunchDarkly project that you want to evaluate, set `LAUNCHDARKLY_FLAG_KEY` to the flag key; otherwise, a boolean flag of `sample-feature` will be assumed.
 
-    public const string FeatureFlagKey = "my-flag";
-```
+    ```bash
+    export LAUNCHDARKLY_SERVER_KEY="1234567890abcdef"
+    export LAUNCHDARKLY_FLAG_KEY="my-boolean-flag"
```

Comment:
Should this be sample-flag?

Reply:
This one is interesting - this is showing the example to override the default (as if they want the `sample-feature` flag, they don't need to change anything.